writer/cassandra/messages.go

```diff
@@ -16,13 +17,68 @@ func NewMessageRepository(session *gocql.Session) writer.MessageRepository {
 	return &msgRepository{session}
 }
 
-func (repo *msgRepository) Save(msg writer.Message) error {
-	cql := `INSERT INTO messages_by_channel
-			(channel, id, publisher, protocol, bn, bt, bu, bv, bs, bver, n, u, v, vs, vb, vd, s, t, ut, l)
-			VALUES (?, now(), ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+func normalize(msg writer.RawMessage) ([]writer.Message, error) {
+	var s, n senml.SenML
```

Comment:
Use group definition:

```
var (
 s, n senml.SenML
 err error
)
```

Also, perhaps is better to rename `s` to `rm` (raw message) and `n` to `nm` (normalized message).